assets/src/blocks/HighlightMatches.js

```diff
@@ -0,0 +1,17 @@
+import { Fragment } from '@wordpress/element';
+
+export const HighlightMatches = ( cellValue, searchText, className = 'highlighted-text' ) => {
+	let reg = new RegExp('(' + searchText.trim() + ')', 'gi')
```

Comment:
Not sure if this is the right place to perform the `trim` on the searchtext. It's not impossible that we actually want to match a trailing whitespace. https://github.com/bvaughn/react-highlight-words also works that way, you can check at https://codesandbox.io/s/5v8yqoxv7k?file=/src/index.js

Reply:
And we can just trim `searchText` once when setting it on the state, shouldn't be needed afterwards.

Reply:
Trimming the value on the state actually prevents you from typing any spaces in the controlled input, is not impossible to match a trailing whitespace but I think it's an edge case, also, as this highlight implementation is just making the text bold, you can't actually see a space being highlighted, so I guess I'll just keep it as is for now, we can adopt that library afterwards if it makes sense... I think we will probably add it at some point...